### ISSUE TITLE
Superscript subscript not working

### ISSUE THREAD

**Issue by sanjeebsarkar** [Aymkdn/html-to-pdfmake#97]
Hi , 
Thank you for making this , it is really helpful.
I noticed that superscript and subscript is not being rendered.

input :
`<p>Normal<sub>Subscript</sub><sup>Supercript</sup></p>`

output :

![image](https://user-images.githubusercontent.com/58333434/113388896-a9f51000-93ac-11eb-99c1-6fdd518fbbf7.png)

Can you please help me with this?

**Comment by Aymkdn** [comment]
This library is based on [PDFMake](https://github.com/bpampuch/pdfmake). Can you tell me how to make it work with PDFMake, please? Then I can tell you how to do it with `html-to-pdfmake`.

**Comment by sanjeebsarkar** [comment]
Hi ,

So here is how I am working with this,

```
const PdfPrinter = require("pdfmake");
const htmlToPdfmake = require("html-to-pdfmake");


let jsdom = require("jsdom");
let { JSDOM } = jsdom;
let { window } = new JSDOM("");
const fs = require('fs');
const font_data = require('./functions/get_fonts.js')

var fonts = font_data.font_list();

  var html = htmlToPdfmake(`<p>Normal<sub>Subscript</sub><sup>Supercript</sup></p>`, {window: window});


  let dd = {
    pageSize: "A4",
    pageOrientation: "portrait",
    content: [
        {
          text:html
          
        }
        
    ]
}


let printer = new PdfPrinter(fonts);
let doc = printer.createPdfKitDocument(dd);
doc.pipe(fs.createWriteStream('D:\\html_document.pdf'));
doc.end();
```

So the above code is generating this output:

![image](https://user-images.githubusercontent.com/58333434/113390292-0b1de300-93af-11eb-8dc0-1006bbc0c90d.png)

Can you please let me know what mistake I am making?

**Comment by Aymkdn** [comment]
This is not my question…

My library translates HTML code to [PDFMake code](https://github.com/bpampuch/pdfmake). So I need to know how the subscript and superscript are done with PDFMake in order to generate the appropriate code (I quickly searched but I didn't find).

So, please, search on the Web how to do it with PDFMake, then share the code with me, and then I can either explain you how to make it work with `html-to-pdfmake` or I can write a fix to make it work.

Does it make sense?

**Comment by sanjeebsarkar** [comment]
Hi ,

Yes it does, give me some time, i will revert back. 
Thank you for ur time.

**Comment by sanjeebsarkar** [comment]
Hi,
this seems to be doing the trick, 
```
var dd = {
	content: [
	    {text: [
           {text:'some',fontSize: 14},
           {text:'sub', sup: { offset: '30%', fontSize: 8 }},
           {text:'text',fontSize: 14}
           ]
        }
	]
	
}
```

**Comment by Aymkdn** [comment]
Thanks, it's what I was looking for. I may add the support for `sub` and `sup`. Let me think about it and I'll get back to you later today, or tomorrow.

**Comment by Aymkdn** [comment]
I've just published v2.1.7 that adds support for `<sup>` and `<sub>` tags

**Comment by sanjeebsarkar** [comment]
Hi,
Thank you so much for making this happen in a short period of time. I will try and let you know.
Thanks again.
